merkle/objhasher/objhasher.go

```diff
@@ -16,42 +16,29 @@
 package objhasher
 
 import (
-	"crypto/sha256" // Use SHA256 to match ObjectHash.
+	"crypto"
 
 	"github.com/benlaurie/objecthash/go/objecthash"
+	"github.com/google/trillian/merkle"
+	"github.com/google/trillian/merkle/maphasher"
```

Comment:
Object hasher depends on maphasher? That's strange if you want to use it with log entries.